modules/effects/src/main/scala/dev/profunktor/redis4cats/redis.scala

```diff
@@ -871,11 +872,11 @@ private[redis4cats] class BaseRedis[F[_]: Concurrent: ContextShift, K, V](
       .futureLift
       .map(Option.apply)
 
-  override def lPush(key: K, values: V*): F[Unit] =
-    async.flatMap(c => F.delay(c.lpush(key, values: _*))).futureLift.void
+  override def lPush(key: K, values: V*): F[Long] =
+    async.flatMap(c => F.delay(c.lpush(key, values: _*))).futureLift.asScala
```

Comment:
You will find we use `.map(x => Long.box(x))` in other functions. I think we can do the same in all the places where you call the new `asScala`.